web/package.json

```diff
@@ -98,7 +98,6 @@
   },
   "devDependencies": {
     "@mdx-js/react": "^2.1.5",
-    "@microsoft/eslint-formatter-sarif": "^3.0.0",
```

Comment:
not used

Reply:
It is used in the CI.
Please re-install it.

Reply:
done